lib/streamlit/elements/lib/policies.py

```diff
@@ -51,9 +52,12 @@ def check_callback_rules(dg: DeltaGenerator, on_change: WidgetCallback | None) -
 def check_session_state_rules(
     default_value: Any, key: str | None, writes_allowed: bool = True
 ) -> None:
-    """Ensures that no values are set for widgets with the given key when writing is not allowed.
+    """Ensures that no values are set for widgets with the given key when writing \
```

Comment:
Is the `\` at the ned of the docstring really needed here? I also remember that there was a problem with putting that into the string (related to the docstring formatting).

Reply:
For docstrings, I think you can just wrap the line, and most tools that parse docstrings won't consider the docstring summary ended until they see a double return.

Reply:
yep good point, its not needed 👍